doc/nrf/ug_nrf7002.rst

```diff
@@ -0,0 +1,265 @@
+
+1. Build the non-secure |NCS| application for the application core using ``-DBOARD=nrf7002dk_nrf5340_cpuapp_ns``.
+   To invoke the building of TF-M, the |NCS| build system requires the Kconfig option :kconfig:option:`CONFIG_BUILD_WITH_TFM` to be enabled, which is done by default when building |NCS| as a non-secure application.
+
+   The |NCS| build system performs the following steps automatically:
+
+      a. Build the non-secure firmware image as a regular |NCS| application.
+      #. Build a TF-M (secure) firmware image.
+      #. Merge the output image binaries.
+      #. Optionally, build a bootloader image (MCUboot).
+
+   .. note::
+      Depending on the TF-M configuration, an application DTS overlay can be required, to adjust the non-secure image flash memory partition and SRAM starting address and sizes.
+
+#. Build the application firmware for the network core using ``-DBOARD=nrf7002dk_nrf5340_cpunet``.
+
+
+Building the secure firmware using |NCS|
+----------------------------------------
+
+To build the secure and the non-secure firmware images using |NCS|, follow these steps:
+
+1. Build the secure |NCS| application for the application core using ``-DBOARD=nrf7002dk_nrf5340_cpuapp``.
+   Also set :kconfig:option:`CONFIG_TRUSTED_EXECUTION_SECURE` to ``y`` and :kconfig:option:`CONFIG_BUILD_WITH_TFM` to ``n`` in the application project configuration file.
+#. Build the non-secure |NCS| application for the application core using ``-DBOARD=nrf7002dk_nrf5340_cpuapp_ns``.
+#. Merge the two binaries.
+#. Build the application firmware for the network core using ``-DBOARD=nrf7002dk_nrf5340_cpunet``.
+
+
+When building a secure and a non-secure application for the nRF5340 application core, the secure application must set the IDAU (SPU) configuration to allow non-secure access to all CPU resources used by the non-secure application firmware.
+SPU configuration must be applied before jumping to the non-secure application.
+
+Building a secure only application
+==================================
+
+Build the |NCS| application following the standard process (see :ref:`gs_programming` for more details), using ``-DBOARD=nrf7002dk_nrf5340_cpuapp`` for the firmware running on the nRF5340 application core and ``-DBOARD=nrf7002dk_nrf5340_cpunet`` for the firmware running on the nRF5340 network core.
+
+Programming the firmware to the DK
+==================================
+
+Follow the instructions in the :ref:`gs_programming` page to build and flash applications as usual.
+
+.. note::
+   To flash and debug applications on the nRF7002 DK you need to upgrade the nRF Command Line Tools to version 10.12.0.
+   For further information on how to install the nRF Command Line Tools, see :ref:`gs_updating`.
+
+Follow the steps in this example to run the :ref:`hello_world` application on the nRF5340 application core:
+
+1. Run your favorite terminal program to listen for output.
+
+   .. code-block:: console
+
+      $ minicom -D <tty_device> -b 115200
```

Comment:
Is this the full command? Isn't there some other setting that should be explicitly declared or set by default?